modules/private-testnet-node.nix

```
@@ -68,6 +68,16 @@ in
       '';
     };
 
+    updateSystemStartTime = lib.mkOption {
+      internal = true;
+      type = lib.types.bool;
+      default = true;
+      description = ''
+        Whether to update the system start time in Shelley and Byron
+        genesis files.
+      '';
+    };
```

```suggestion
    updateSystemStartTime = lib.mkEnableOption "updating the system start time in Shelley and Byron genesis files" // {
      default = true;
    };
```

This should be equivalent, the final description is what you pass to `mkEnableOption` prefixed with `Wheter to enable`.